Update USA holiday calendar to 2020-2030

justdave opened the pull request:
Fixes #221

Sancus commented:
check_calendars.py dies on both calendar PRs with:
```
Traceback (most recent call last):
  File "l10n_tools/check_calendars.py", line 44, in <module>
    run()
  File "l10n_tools/check_calendars.py", line 34, in run
    check_if_correct_parse(f)
  File "l10n_tools/check_calendars.py", line 19, in check_if_correct_parse
    cal_entries = Calendar.from_ical(fh.read(), multiple=True)
  File "/usr/lib/python2.7/site-packages/icalendar/cal.py", line 360, in from_ical
    component = stack.pop()
IndexError: pop from empty list
```

Not sure if the .ics files are bugged or if the python script is bugged, but it doesn't produce this error with any of the old files.

Sancus reviewed (changes requested):
The USHolidays.ics file breaks l10n_tools/check_calendars.py and I'm not quite sure why. The other files are fine. The file does import into Thunderbird though. Maybe importing into TB and then exporting again might fix it.

justdave commented:
Turns out Thunderbird doesn't like the calendar either. It just silently ignores the error and bails without telling the user after importing what it got up to the broken part.

There was a duplicate END:VEVENT tag in the file on the end of the Election Day event. It essentially truncated on import at that spot.

I've also renamed the category tag on all of the events from `"Holiday"` to `Holidays` to match the built-in category in Thunderbird, and removed trailing spaces from a bunch of the event names and descriptions.
